Makefile.js

```diff
@@ -32,7 +32,10 @@ const NODE_MODULES = "./node_modules/",
     MAKEFILE = "Makefile.js",
     CONFIG_FILES = ".eslintrc.js",
     JS_FILES = "\"lib/**/*.js\" \"espree.js\"",
-    TEST_FILES = "tests/lib/**/*.js";
+    TEST_FILES = "tests/lib/**/*.js",
+
+    // FLAGS
+    CLIFLAGS = process.argv.slice(3);
```

Comment:
Looking at the documentation, I’m not sure `nodeCLI.exec()` accepts an array as an argument. Should this be the following?

```suggestion
    CLIFLAGS = process.argv.slice(3).join(“ “);
```

Alternatively, we could use the spread operator in the function calls below:

```js
lastReturn = nodeCLI.exec("eslint", MAKEFILE, ...CLIFLAGS);
```

Reply:
I am a bit confused as in [this line](https://github.com/nzakas/shelljs-nodecli/blob/master/lib/shelljs-nodecli.js#L76) it does say that the args are converted an array but [here](https://github.com/nzakas/shelljs-nodecli/blob/master/lib/shelljs-nodecli.js#L90) it is looping the argument.

I think its better to use the spread operator.